lib/index.js

```diff
@@ -2383,7 +2388,7 @@ addMethod('zipEach', function (ys) {
  */
 
 addMethod('zip', function (ys) {
-    return this.create([this, this.create(ys)]).zipAll();
+    return this.createChild(this.create([this, this.create(ys)]).zipAll());
```

Comment:
I this the correct thing to do? There's an extra layer of wrapping here that's not necessary.

I think this can be implemented simply with `this.create([this, this.create(ys)]).zipAll();`